src/asgen/handlers/iconhandler.d

```diff
@@ -129,6 +136,9 @@ public:
 
     private bool directoryMatchesSize (Algebraic!(int, string)[string] themedir, ImageSize size)
     {
+        int scale = themedir["scale"].get!(int);
```

Comment:
You could make this just `themedir["scale"].get!int;`, D doesn't need the brackets here (that particular code is rather old from when I didn't know that much about D yet ^^)
But that's just a style-thing, no need to change it.